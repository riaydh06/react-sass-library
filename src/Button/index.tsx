import React from "react";

interface ButtonProps {
  label: string;
}

const Button = ({ label }: ButtonProps) => {
  return <button className="p-5">{label}</button>;
};

export default Button;
