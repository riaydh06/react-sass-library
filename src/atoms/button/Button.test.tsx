import React from 'react';
import renderer from 'react-test-renderer';

import Button from './index';

it('renders correctly', () => {
  const tree = renderer.create(<Button label="Test Button" />).toJSON();
  expect(tree).toMatchSnapshot();
});
