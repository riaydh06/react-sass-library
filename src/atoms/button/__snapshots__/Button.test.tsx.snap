// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="p-5"
>
  Test Button
</button>
`;
